website/docs/sql/scalar_func/datetime.md

```diff
@@ -88,6 +88,8 @@ Function|Description|Example
 `quarter(timestamp)` → `integer`|Get quarter (equivalent to `EXTRACT(DAY FROM ...)`)|`quarter(timestamp '2001-02-16 20:38:40')` → `1`
 `second(timestamp)` → `numeric`|Get second (equivalent to `EXTRACT(SECOND FROM ...)`)|`second(timestamp '2001-02-16 20:38:40.500')` → `40.500`
 `second (interval)` → `numeric`|Get second (equivalent to `EXTRACT(SECOND FROM ...)`)|`second(interval '21 seconds')` → `21.0`
```

Comment:
please only merge this PR after https://github.com/tableau/hyper-db/pull/20 got merged